docs/cli-reference.md

```diff
@@ -1 +1,228 @@
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | trip_distance         | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | RatecodeID            | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | store_and_fwd_flag    | String    | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | PULocationID          | Int       | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | DOLocationID          | Int       | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | payment_type          | Int       | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | fare_amount           | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | extra                 | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | mta_tax               | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | tip_amount            | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | tolls_amount          | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | improvement_surcharge | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | total_amount          | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | congestion_surcharge  | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> |            |       | | airport_fee           | Float     | true     | false | |
+> |            |       | +-----------------------+-----------+----------+-------+ |
+> +------------+-------+----------------------------------------------------------+
+> ```
+
+#### `dozer security generate-token`
+If you want to utilize api security features, run following command to generate the token. Make sure you add following part in the dozer configuration file.
+```yaml
+api:
+  api_security:
+    !Jwt
+```
+
```

Comment:
Add: for mode details on how.to.configure authentication click here ...and we link.to the Api security section